SlicerCART/documentation/installation.md

```diff
@@ -0,0 +1,71 @@
+# SlicerCART Installation
+
+## Requirements
+
+* MacOS Sonoma or Sequoia (15.0.1) is recommended 
+* A working version of [3D Slicer](https://download.slicer.org) (version 5.6.2).
+* Qt: might need to be installed.
+  * The first five steps of the following procedure might be useful for this: [procedure](https://web.stanford.edu/dept/cs_edu/resources/qt/install-mac).
+  * You can try importing Qt in the Slicer python console (e.g. `import slicer ` then `from slicer.util import Qt`)
+
+*Although it may work on other versions and/or operating system, please note 
+that it has not been tested.
+
+If 3D Slicer has not been already installed, you can follow these steps: 
+1. Install [3D Slicer](https://download.slicer.org):
+2. Make sure that you are able to open and use the 3D Slicer software before 
+   trying installing any extension/module. 
+3. If you encounter some problems, you are encouraged to refer to:
+   * [3D Slicer Documentation](https://slicer.readthedocs.io/en/latest/)
+   * [3D Slicer forum](https://discourse.slicer.org/) (very active community)
+
+## Installation Steps
+
+1. Clone the [SlicerCART repository](https://github.com/neuropoly/slicer-manual-annotation) in the location of your choice.
```

Comment:
Turns out slicedCARD doesn't work if not installed in the root directory (tried to copy it on my desktop so I would suggest this new version. So I would definitely not put in the location of your choice.

Reply:
Ok. Hm thats a problem. On my side, it is installed in the location of my choice (not on my desktop --- somewhere in documents etc.) Also, I would avoid the recomandation to use terminal: here, targeted audience includes non-technical staff (e.g. clinicians) for which the word 'terminal' sounds nothing... will keep the discussion open so we can discuss about this point

Reply:
maybe say download instead of cloning, and say how to download, just linking to the repository doesn't help:
![image](https://github.com/user-attachments/assets/1f41ae6b-0db9-4a11-bbbb-00a951a8dbad)
or link to a releas or dowload link: [Download ZIP](https://github.com/neuropoly/slicer-manual-annotation/archive/refs/heads/main.zip)

Reply:
Agree with @Sandrine . but how to link  to the zip download as you did? also to keep the latest version always so no need to update very time?

Reply:
that discussion thread should go in a specific issue